symphony/content/content.publish.php

```diff
@@ -37,33 +37,31 @@ public function sort(&$sort, &$order, $params)
             EntryManager::setFetchSorting($sort, $order);
         } else {
             $sort = General::sanitize($sort);
+            $filters = '';
 
             // Ensure that this field is infact sortable, otherwise
             // fallback to IDs
             if (($field = FieldManager::fetch($sort)) instanceof Field && !$field->isSortable()) {
                 $sort = $section->getDefaultSortingField();
             }
 
+            // Format the filter query string
+            if ($params['filters']) {
+                $filters = preg_replace('/^&amp;/i', '', $params['filters'], 1);
```

Comment:
Wouldn't it be simpler, faster and more bulletproof to use `ltrim($params['filters'], '&amp;')`?